views/macros/default-value.njk

```diff
@@ -2,8 +2,8 @@
   - `property`: form value to look up
 #}
 
-{% macro defaultValue(property) %}{% if data[property] %}{{ data[property] }}{% else %}
+{% macro defaultValue(property) %}
+  {% if (property in data) and (data[property] !== "")  %}
+    {{ data[property] }}{% else %}
     <span style="color:red; font-weight:bold">
```

Comment:
Ah, so if you don't write the macros on one line you will get spaces around them in the agreement
(I should definitely have put a comment there explaining this)
<img width="206" alt="image" src="https://user-images.githubusercontent.com/8228248/66572623-4b483080-eb3f-11e9-8bf1-bf75ccdd71a8.png">

🙃